bedrock/security/templates/security/bug-bounty/web-eligible-sites.html

```diff
@@ -232,26 +168,22 @@ <h3>Private Relay</h3>
     <li>relay.firefox.com</li>
   </ul>
 
-  <h3>Ship It</h3>
-  <ul class="mzp-u-list-styled">
-    <li>shipit-api.mozilla-releng.net</li>
-    <li>shipit.mozilla-releng.net</li>
-  </ul>
-
   <h3>Source Control</h3>
   <ul class="mzp-u-list-styled">
     <li>hg.mozilla.org (website only)</li>
   </ul>
 
-  <h3>Speak To Me</h3>
+  <h3>VPN</h3>
   <ul class="mzp-u-list-styled">
-    <li>speaktome-2.services.mozilla.com</li>
-    <li>speaktome.services.mozilla.com</li>
+    <li>vpn.mozilla.org (VPN client)</li>
   </ul>
 
-  <h3>VPN</h3>
+  <h3>Support</h3>
   <ul class="mzp-u-list-styled">
-    <li>vpn.mozilla.org</li>
+    <li>support.mozilla.org</li>
   </ul>
+  <p>
+    <br> Please use our <a href="support.allizom.org">staging instance</a> for testing to avoid disrupting users.
+  </p>
```

Comment:
Good idea on the note! Maybe we can add similar for bugzilla.mozilla.org (but that's harder, because the allizom version is a snapshot instance with new account creation disabled)

Reply:
there is a similar note about bugzilla, I got inspiration from there 🤓